### PR TITLE
feat: `countWords`

### PR DESCRIPTION
**Pull request by protob**
Please merge feature countWords

### CODE REVIEW

File: packages/countWords/README.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+```javascript
+import countWords from '@plexis/count-words';
+
+countWords();
+// => 0
+countWords('');
+// =>  0
+countWords('     ');
+// => 2
```
Comment (AngelGzS) [inline packages/countWords/README.md]: Use one line example, like this 

`countWords('     '); // returns  0
`

File: packages/countWords/README.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+`npm i @plexis/count-words`
+
+**Usage**
+
+```javascript
+import countWords from '@plexis/count-words';
+
+countWords(); // returns 0
+countWords(''); // returns  0
+countWords('     '); // returns 2
```
Comment (AngelGzS) [inline packages/countWords/README.md]: Should be

`countWords('     '); // returns 0`

Comment (protob) [reply packages/countWords/README.md]: done

**Comment by protob** [comment]
What about "wontfix" issues? How much time has to pass before they close?

**Comment by papb** [comment]
> 
> 
> This issue has been automatically marked as stale because it has not had recent activity. It will be closed if no further activity occurs. Thank you for your contributions. May the force be with you!

:eyes:

**Comment by papb** [comment]
> This issue has been automatically marked as stale because it has not had recent activity. It will be closed if no further activity occurs. Thank you for your contributions. May the force be with you!

:eyes:

**Comment by papb** [comment]
> 
> 
> This issue has been automatically marked as stale because it has not had recent activity. It will be closed if no further activity occurs. Thank you for your contributions. May the force be with you!

:eyes:

**Comment by protob** [comment]
@vorillaz, Could you merge this feature please?

**Comment by papb** [comment]
> 
> 
> This issue has been automatically marked as stale because it has not had recent activity. It will be closed if no further activity occurs. Thank you for your contributions. May the force be with you!

:eyes:

**Comment by papb** [comment]
> 
> 
> This issue has been automatically marked as stale because it has not had recent activity. It will be closed if no further activity occurs. Thank you for your contributions. May the force be with you!

:eyes:

**Comment by papb** [comment]
> This issue has been automatically marked as stale because it has not had recent activity. It will be closed if no further activity occurs. Thank you for your contributions. May the force be with you!

:eyes:

**Comment by papb** [comment]
> This issue has been automatically marked as stale because it has not had recent activity. It will be closed if no further activity occurs. Thank you for your contributions. May the force be with you!

:eyes:

**Comment by papb** [comment]
> This issue has been automatically marked as stale because it has not had recent activity. It will be closed if no further activity occurs. Thank you for your contributions. May the force be with you!

:eyes:

**Comment by papb** [comment]
> This issue has been automatically marked as stale because it has not had recent activity. It will be closed if no further activity occurs. Thank you for your contributions. May the force be with you!

:eyes:

**Comment by papb** [comment]
> This issue has been automatically marked as stale because it has not had recent activity. It will be closed if no further activity occurs. Thank you for your contributions. May the force be with you!

:eyes:

**Comment by papb** [comment]
> 
> 
> This issue has been automatically marked as stale because it has not had recent activity. It will be closed if no further activity occurs. Thank you for your contributions. May the force be with you!

:eyes:

**Comment by papb** [comment]
:eyes:

**Comment by papb** [comment]
Oh no I lost to stalebot

**Comment by vorillaz** [comment]
Sorry for that @papb , merging as we speak.

**Comment by papb** [comment]
Thank you!

**Pull request merged**